src/services/action.service.ts

```diff
@@ -28,6 +30,28 @@ export class ActionService {
         }
     }
 
+    saveJob(job: JobsConfig) {
+        let jobsConfig = require('../../config/jobs/jobs.config.json');
+        jobsConfig.push(job);
+        let newJobs = JSON.stringify(jobsConfig);
+        let s = __dirname + '/../../config/jobs/jobs.config.json';
```

Comment:
Put this path in a constant, so in case we change it, we don't have to replace everywhere
This also solves the problem in case we want to configure an external directory, let's say: process.env.CONFIG_LOCATION